cloud/store/service.go

```diff
@@ -38,16 +40,28 @@ type client interface {
 }
 
 func NewService(store *Store, client client) *Service {
-	return &Service{store: store, cl: client}
+	return &Service{store: store, cl: client, log: store.log}
 }
 
 // / RAFT-TODO Documentation
 func (s *Service) Open(ctx context.Context, db Indexer) error {
+	s.log.Info("bootstrapping started")
```

Comment:
thought: I think we should keep the `bootstrapping` terminology only for the `bootstrapper` code/struct. Here we are just opening the store, so maybe `starting raft` or `attempting to start raft` would be clearer.

Reply:
Sure we can modify the message to improve clarity